Day1/C/fizzbuzz2.c

```diff
@@ -0,0 +1,29 @@
+/*
+        * @author: shashanknenar
```

Comment:
Kindly fix the indentation for the comments in both the source code file as well as the README